### ISSUE TITLE
Why does BrowserConsoleHandler always use console.log?

### ISSUE THREAD

**Issue by pafernandez-oesia** [Seldaek/monolog#1728]
Hi!

I was just wondering if there is any reason to use `console.log` method for all log levels, instead of also using `debug`, `info`, `warn` and `error` methods depending on the log level. This makes filtering output easier in the browser console and clearly highlights error level messages.

The following image shows an example output from Laravel using Monolog and a custom implementation of `BrowserConsoleHandler`:

![image](https://user-images.githubusercontent.com/96843912/177963365-5fa14776-1666-40c4-9a6c-909d36b0e68d.png)

I can make a PR if you find it useful.

Kind regards,

**Comment by Seldaek** [comment]
Yeah that sounds good, I think it's mostly because this was done when browser consoles were still pretty basic :)
